tools/AduCmdlets/AduAzStorageBlobHelper.psm1

```diff
@@ -0,0 +1,80 @@
+#
+# Azure Device Update for IoT Hub
+# Powershell module for uploading file to Azure Storage Blob Container.
+# Copyright (c) Microsoft Corporation.
+#
+
+#Requires -Version 5.0
+#Requires -Modules Az.Accounts, Az.Storage
+
+function Get-AduAzBlobContainer
+{
+<#
+    .SYNOPSIS
+        Get a reference to Azure Storage Blob container object.
+#>
+    [CmdletBinding()]
+    Param(
+        # Azure subscription id.
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $SubscriptionId,
+
+        # Azure resource group name of the Azure Storage account.
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $ResourceGroupName,
+
+        # Azure Storage account name.
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $StorageAccountName,
+
+        # Blob container name.
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $ContainerName,
+
+        # Skip connecting to Azure.
+        [switch] $SkipLogin
```

Comment:
what does this do?  Is the assumption that there's a cached connection already?  If so, wouldn't 44-46 be a no-op?

Reply:
Skips `connect-azaccount` call. For e.g. to get ref to 2nd container, you can append -SkipLogin. 

`connect-azaccount` doesn't know to cache anything, hence this switch.